package/gluon-mesh-batman-adv-core/files/lib/gluon/upgrade/310-gluon-mesh-batman-adv-core-mesh

```diff
@@ -51,9 +51,22 @@ uci:section('network', 'interface', 'bat0',
 		    proto = 'none',
 		    macaddr = sysconfig.primary_mac,
 		    multicast_router = 2,
+		    learning = 0,
 	    }
 )
 
+uci:delete('network', 'client_lan')
+if sysconfig.lan_ifname then
+	uci:section('network', 'interface', 'client_lan',
+		    {
+			    unicast_flood = 0,
+		    }
+	)
+	for _, lanif in ipairs(lutil.split(sysconfig.lan_ifname, ' ')) do
+		uci:set('network', 'client_lan', 'ifname', lanif)
+	end
```

Comment:
Hm, your two suggestions create a "list ifname" but a "list ifname eth0.1" for instance does not seem to work, unicast_flood is not set correctly upon reboot. Only an "option" works.

Is there supposed to ever be more than one lan interface?